lib/home_page.dart

```diff
@@ -38,76 +36,107 @@ class _HomePageState extends State<HomePage> {
+                    children: [
+                      ElevatedButton(
+                        style: ElevatedButton.styleFrom(
+                          primary: Colors.red.shade700,
+                        ),
+                        onPressed: pageNumber != 1
+                            ? () {
+                                _pokemonBloc.getPreviousPage();
+                                pageNumber--;
+                                setState(() {});
+                              }
+                            : () {},
+                        child: Icon(
+                          Icons.arrow_back,
+                          color: Colors.black,
+                        ),
+                      ),
+                      snapshot.data?.state == StateType.loading
+                          ? Text('')
+                          : Stack(
+                              children: [
+                                Text(
+                                  StringConstants.pageText +
+                                      ' $pageNumber - $cantPages',
+                                  style: TextStyle(
+                                    fontFamily: 'Pokemon',
+                                    fontSize: NumericConstants.pageTextFontSize,
+                                    foreground: Paint()
+                                      ..style = PaintingStyle.stroke
+                                      ..strokeWidth = NumericConstants
+                                          .pageForegroundStrokeWidth
+                                      ..color = firstPageColor,
+                                  ),
+                                ),
+                                Text(
+                                  StringConstants.pageText +
+                                      ' $pageNumber - $cantPages',
+                                  style: TextStyle(
+                                    fontFamily: 'Pokemon',
+                                    fontSize: NumericConstants.pageTextFontSize,
+                                    color: secondPageColor,
+                                  ),
+                                ),
+                              ],
+                            ),
+                      ElevatedButton(
+                        style: ElevatedButton.styleFrom(
+                          primary: Colors.red.shade700,
+                        ),
+                        onPressed: pageNumber != cantPages
+                            ? () {
+                                _pokemonBloc.getNextPage();
+                                pageNumber++;
+                                setState(() {});
+                              }
+                            : () {},
+                        child: Icon(
+                          Icons.arrow_forward,
+                          color: Colors.black,
+                        ),
+                      ),
```

Comment:
Here uses the same widget mentioned in the previous comment

Reply:
Fixed